include/unifex/completion_channels.hpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Rishabh Dwivedi <rishabhdwivedi17@gmail.com>
```

Comment:
Please use the text from [LICENSE.txt](https://raw.githubusercontent.com/facebookexperimental/libunifex/main/LICENSE.txt)